script/1_0_2current_en_script/Common/00_02.txt

```diff
@@ -1,23 +1,23 @@
 <Page1>
 As everyone sleeps, my eyes open to the night's cold air. 
 It's pitch black tonight.
-There is no one in the house.
+There's no one in the house.
 <Page2>
-The domestic smell of tatami mats and shoji paper
+The domestic smell of tatami mats and shoji paper.
 A sky inlaid with a parquet of stars.
 The specters of shadows hang from the corners of the room like sludge.
 A silence devoid of a single note.
 Not even the familiar sound of rustling clothes.
 <Page3>
 I stand completely still, idly waiting for anyone.
-In the sound of extinction, I remember the past.
+Within the sound of extinction, I remember the past.
```

Comment:
I think when I edited this line I wasn't paying attention to the JP because this is straight up mis-translated.
死に絶えた音の中で過去を偲ぶ
> In the dead silence, I remember the past.